Doc/library/http.client.rst

```diff
@@ -416,6 +424,14 @@ HTTPConnection Objects
    .. versionadded:: 3.7
 
 
+.. attribute:: HTTPConnection.max_response_headers
+
+   The maximum number of allowed response headers to help prevent denial of
```

Comment:
You should indicate what its default would be if we were to pass None to the constructor.